### PR TITLE
create a new PR "that has everything" needed change

### PR DESCRIPTION
**Pull request by masayyad**
what should i do to**bold**, _italic_,
 
image

### CODE REVIEW

**Comment by masayyad** [comment]
adding <new> "hello" and many more

File: new file [hunk 1]
```diff
@@ -9,7 +9,7 @@ blah blah
 
 
 hey
```
Comment (masayyad) [inline new file:11]: new comment "hello" and many more

**Review by mahanoor786 — COMMENTED** [review]
adding comment "quotes", 'single quotes', and much more

**Review by masayyad — COMMENTED** [review]
add a review comment "double", 'single' and much more

**Review by masayyad — COMMENTED** [review]
adding comment "double", 'single' and much more 
image

**Comment by masayyad** [comment]
adding a comment

**Comment by masayyad** [comment]
add comment

File: new file [hunk 2]
```diff
@@ -9,7 +9,7 @@ blah blah
```
Comment (masayyad) [inline new file:9]: comment

File: new file (hunk 1 above)
Comment (masayyad) [reply new file:11]: comm

Comment (masayyad) [reply new file:11]: add a comment

**Comment by masayyad** [comment]
add some comment

**Comment by masayyad** [comment]
add comment

File: new file (hunk 1 above)
Comment (masayyad) [reply new file:11]: add

**Review by mahanoor786 — COMMENTED** [review]
add comwmnt

**Comment by masayyad** [comment]
qad ccc

File: new file (hunk 2 above)
Comment (masayyad) [reply new file:9]: add

**Review by mahanoor786 — COMMENTED** [review]
add

**Review by mahanoor786 — COMMENTED** [review]
adding _referral_ "ffff" <gg>

**Review by mahanoor786 — APPROVED** [review]
approve

**Review by mahanoor786 — COMMENTED** [review]
adding comment chnage

**Review by mahanoor786 — COMMENTED** [review]
adding comment chnages

**Review by mahanoor786 — APPROVED** [review]
checking approval

**Review by mahanoor786 — CHANGES_REQUESTED** [review]
checking request chnages

**Comment by mahanoor786** [comment]
add comemnt

**Pull request closed without merging**